priiloader/hacks_hash.ini

```diff
@@ -314,3 +314,12 @@ minversion=1
 amount=1
 hash=0x68747470,0x733a2f2f,0x25732f6e,0x75732f73
 patch=0x6E756C6C
+[480p graphics fix in system menu]
+maxversion=4610
+minversion=448

```

Comment:
you were only able to check versions >= 448?

Reply:
Yeah, I only checked the code for 4.1+. Are 4.0 and below still used by many people? If needed, I could download and check earlier versions as well.

Reply:
i don't think anyone uses anything < 4.1 either, was just wondering. the hash seems to be there in 2.0 as well. just not sure if its also the video init code (but i think it is)

Reply:
If the whole hash is in other versions as well, it should definitely be the video init code. 
I do know that there are two different versions of the video init code, but if you found exactly that hash in 2.0 then apparently they always used the same one in the system menu. If it's present in 2.0, we could as well set minversion to 97.

Reply:
then lower the min version, and i'll merge it.